pkg/server/secrets-encrypt.go

```diff
@@ -12,16 +12,24 @@ import (
 	"strings"
 	"time"
 
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+
```

Comment:
nit: remove trailing empty line so that this gets sorted in with the rest of the imports
```suggestion
	apierrors "k8s.io/apimachinery/pkg/api/errors"
```